Rakefile

```diff
@@ -19,7 +19,10 @@ namespace :style do
     FoodCritic::Rake::LintTask.new(:chef) do |t|
       t.options = {
         fail_tags: ['any'],
-        progress: true
+        progress: true,
+        tags: %w(
+          ~solo
```

Comment:
It's better to stick this same line in a .foodcritic file so no matter how foodcritic is run it'll get ignored